frontend/src/pages/AmbassadorsPageBO.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function AmbassadorsPageBO() {
+  return <div></div>;
```

Comment:
These pages are a bit empty for now :p 
I'd have created the AdminNavbar but not added the links for now